NitroxClient/GameLogic/ChatUI/PlayerChatManager.cs

```diff
@@ -96,22 +97,20 @@ public void SendMessage()
 
         private IEnumerator LoadChatLogAsset()
         {
-            yield return AssetBundleLoader.LoadUIAsset("chatlog", true, playerChatGameObject =>
-            {
-                playerChat = playerChatGameObject.AddComponent<PlayerChat>();
-            });
-
+            yield return LoadUIAsset(NitroxAssetBundle.CHAT_LOG, true);
+            
+            GameObject playerChatGameObject = (GameObject)NitroxAssetBundle.CHAT_LOG.LoadedAssets[0];
+            playerChat = playerChatGameObject.AddComponent<PlayerChat>();
+            
             yield return playerChat.SetupChatComponents();
         }
 
-        public void LoadChatKeyHint()
+        public IEnumerator LoadChatKeyHint()
         {
             if (!NitroxPrefs.ChatUsed.Value)
```

Comment:
```suggestion
            if (!NitroxPrefs.ChatUsed.Value || true)
```
For testing purposes